runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -270,6 +266,31 @@ BaseLoader<LoaderDomain, SpecificLoader>::BaseLoader::tensorIdxToOperandIdx(int3
   return isOptionalInputTensor(tensorIdx) ? ir::OperandIndex() : _tensor_to_operand[tensorIdx];
 }
 
+template <typename LoaderDomain, typename SpecificLoader>
+void BaseLoader<LoaderDomain, SpecificLoader>::BaseLoader::deallocateMmappedArea(uint8_t *ptr,
+                                                                                 size_t size,
+                                                                                 int32_t pagesize)
+{
+  // Calculate offset from base address of mapped region
+  auto unaligned_offset_start = ptr - _base;
```

Comment:
Personally, `auto` keyword has been used too much, I think. For me, `unaligned_offset_start` could be `uint8_t*` or `uint32_t`. Below `aligned_offset_start` is calculated by division and multiplication. If so that could be `double`. (If that is your intention, ignore my comment.)

In short, how about using the exact type instead of `auto`(type deduction)?

Reply:
@YongseopKim I fount that pointer subtraction should use `ptrdiff_t` type. https://en.cppreference.com/w/cpp/types/ptrdiff_t.
I used auto keyword here because I don't know correct type for `ptrdiff_t` + `size_t`.
I'll find out correct type and use exact type here.